### ISSUE TITLE
After upgrade to 2.0.1, APK size gained 1.6Mb

### ISSUE THREAD

**Issue by peterpeterparker** [ionic-team/ionic-app-scripts#1103]
#### Short description of the problem:

After my upgrade to 2.0.1, my produced APK size gained 1.6Mb. My APK used to be 7.5mb, now 9.1mb

#### What behavior are you expecting?

At least the same size has before aka 1.6mb less

**Steps to reproduce:**
1. Upgrade to app-scripts 2.0.1
2. ionic cordova build android --prod

**Which @ionic/app-scripts version are you using?**

2.0.1

**Other information:** (e.g. stacktraces, related issues, suggestions how to fix, stackoverflow links, forum links, etc)

global packages:

    @ionic/cli-utils : 1.4.0
    Cordova CLI      : 7.0.1 
    Gulp CLI         : CLI version 1.2.2 Local version 3.9.1
    Ionic CLI        : 3.4.0

local packages:

    @ionic/app-scripts              : 2.0.1
    @ionic/cli-plugin-cordova       : 1.4.1
    @ionic/cli-plugin-gulp          : 1.0.2
    @ionic/cli-plugin-ionic-angular : 1.3.2
    Cordova Platforms               : android 6.2.3 ios 4.4.0
    Ionic Framework                 : ionic-angular 3.5.0

System:

    Node       : v8.1.3
    OS         : macOS Sierra
    Xcode      : Xcode 8.3.3 Build version 8E3004b 
    ios-deploy : 1.9.0 
    ios-sim    : 5.0.13 
    npm        : 5.1.0

**Comment by basvdijk** [comment]
Got the same problem went from 5.7MB to 7MB

```
global packages:

    @ionic/cli-utils : 1.5.0
    Cordova CLI      : 6.5.0
    Ionic CLI        : 3.5.0

local packages:

    @ionic/app-scripts              : 2.0.1
    @ionic/cli-plugin-cordova       : 1.4.1
    @ionic/cli-plugin-ionic-angular : 1.3.2
    Cordova Platforms               : android 6.1.2 ios 4.3.1
    Ionic Framework                 : ionic-angular 3.5.0

System:

    Node       : v7.2.0
    OS         : OS X El Capitan
    Xcode      : Xcode 8.2.1 Build version 8C1002
    ios-deploy : not installed
    ios-sim    : 5.0.13
    npm        : 5.1.0
```

**Comment by biesbjerg** [comment]
Probably the core issue: https://github.com/ionic-team/ionic-app-scripts/issues/1104

**Comment by gianpaj** [comment]
same here. mine got 1.76 Mb larger!

**Comment by peterpeterparker** [comment]
thx for pointing the deeptree.js issue @biesbjerg 
agree with you, could be the reason, just tested and I also have a remaining `deptree.js` in my `www/build` when doing a `--prod`build

**Comment by peterpeterparker** [comment]
Issue is solved with #1104 published with version 2.0.2
